docs/main/create-ocp-project/README.md

```diff
@@ -14,97 +14,189 @@ The following table lists the configurable parameters of the Create OCP project
+variable values.
+
+> [!NOTE]
+> Updating the Secret does not automatically restart the workflow Pod, nor update the associated Sonataflow CR. If you update the keys in the secret you must update the Sonataflow CR - as shown below - to reference them. If you update the values in the Secret you must delete the existing `create-ocp-project` workflow Pod to create a new Pod that uses the new values.
+
+> [!WARNING]
+> If you run the `helm upgrade` command, the values of the Secret are reset.
+
+Run the following command to update the Secret. Replace the example values with
+the correct values for your environment:
 
-#### Environment variables
-#### Secret
+```bash
+export TARGET_NS='sonataflow-infra'
+export WORKFLOW_NAME='create-ocp-project'
 
-We also need to set the following environment variables:
-* JIRA_API_TOKEN
-* OCP_API_SERVER_TOKEN
+export NOTIFICATIONS_BEARER_TOKEN='token_from_backstage_appconfig'
+export BACKSTAGE_NOTIFICATIONS_URL='https://backstage.replace-me.com'
 
-To do so, edit the secret `${WORKFLOW_NAME}-creds` and set those values and the one of `NOTIFICATIONS_BEARER_TOKEN`:
+export JIRA_API_TOKEN='token_for_jira_api'
+export JIRA_URL='https://replace-me.atlassian.net/'
+export JIRA_USERNAME='foo@bar.com'
+
+export OCP_API_SERVER_URL='https://api.cluster.replace-me.com:6443'
+export OCP_API_SERVER_TOKEN=$(oc create token orchestrator-ocp-api)
+export OCP_CONSOLE_URL='replaceme'
 ```
-WORKFLOW_NAME=create-ocp-project
-oc -n sonataflow-infra patch secret "${WORKFLOW_NAME}-creds" --type merge -p '{
-   "data":{
-      "NOTIFICATIONS_BEARER_TOKEN":"'$(oc get secrets -n rhdh-operator backstage-backend-auth-secret -o go-template='{{ .data.BACKEND_SECRET  }}')'"
-   },
-   "stringData":{
-      "JIRA_API_TOKEN":"{{ JIRA_API_TOKEN }}",
-      "OCP_API_SERVER_TOKEN":"{{ OCP_API_SERVER_TOKEN }}"
-   }
-}'
+
+If you've installed RHDH using the Orchestrator Operator you can obtain the
+`NOTIFICATIONS_BEARER_TOKEN` using the following command:
+
+```bash
+export NOTIFICATIONS_BEARER_TOKEN=$(oc get secrets -n rhdh-operator backstage-backend-auth-secret -o go-template='{{ .data.BACKEND_SECRET  }}' | base64 -d)
 ```
-If you are using Jira cloud, you can generate the `JIRA_API_TOKEN` using https://id.atlassian.com/manage-profile/security/api-tokens 
 
-The `OCP_API_SERVER_TOKEN` should be associated with a service account.
+Now, patch the Secret with these values:
+
+```bash
+oc -n $TARGET_NS patch secret "$WORKFLOW_NAME-creds" \
+  --type merge -p "{ \
+    \"stringData\": { \
+      \"NOTIFICATIONS_BEARER_TOKEN\": \"$NOTIFICATIONS_BEARER_TOKEN\" \
+      \"JIRA_API_TOKEN\": \"$JIRA_API_TOKEN\", \
+      \"OCP_API_SERVER_TOKEN\": \"$OCP_API_SERVER_TOKEN\", \
+      \"BACKSTAGE_NOTIFICATIONS_URL\": \"$BACKSTAGE_NOTIFICATIONS_URL\", \
+      \"JIRA_URL\": \"$JIRA_URL\", \
+      \"JIRA_USERNAME\": \"$JIRA_USERNAME\", \
+      \"OCP_API_SERVER_URL\": \"$OCP_API_SERVER_URL\", \
+      \"OCP_CONSOLE_URL\": \"$OCP_CONSOLE_URL\" \
+    } \
```

Comment:
There is no need for the `\` inside the json string in the end of the line:
```
... -p "{ 
    \"stringData\": { 
      \"NOTIFICATIONS_BEARER_TOKEN\": \"$NOTIFICATIONS_BEARER_TOKEN\",
      \"JIRA_API_TOKEN\": \"$JIRA_API_TOKEN\", 
      \"OCP_API_SERVER_TOKEN\": \"$OCP_API_SERVER_TOKEN\", 
      \"BACKSTAGE_NOTIFICATIONS_URL\": \"$BACKSTAGE_NOTIFICATIONS_URL\", 
      \"JIRA_URL\": \"$JIRA_URL\", 
      \"JIRA_USERNAME\": \"$JIRA_USERNAME\", 
      \"OCP_API_SERVER_URL\": \"$OCP_API_SERVER_URL\", 
      \"OCP_CONSOLE_URL\": \"$OCP_CONSOLE_URL\" 
    }
  }"
```
works too